app/views.py

```diff
@@ -79,5 +85,93 @@ def add_to_cart(request):
 
         cart_item.save()
 
-        return JsonResponse({"status": "success"})
-    return JsonResponse({"status": "failed"}, status=400)
+        return JsonResponse({"status": _("success")})
+    return JsonResponse({"status": _("failed")}, status=400)
+
+
+@csrf_exempt
+def update_cart(request):
+    try:
+        data = json.loads(request.body)
+        user = User.objects.first()
+        cart, created = Cart.objects.get_or_create(user=user)
+
+        if request.method == "POST":
+            for item in data:
+                item_id = item["id"]
+                quantity = item["quantity"]
```

Comment:
có quản lý số lượng sp còn lại trong bảng sp ko em nhỉ? nếu có thì cần thêm logic để check nếu sp hết nhé

Reply:
em chỉ thêm vào cart thôi, khi người dùng thanh toán để tạo order thì mới thông báo nếu hết sản phẩm ạ